Projects/TEAM-3/Mykhailenko/QA2022_04_401/ex401/LoginTest.cs

```diff
@@ -0,0 +1,45 @@
+using NUnit.Framework;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+
+namespace ex401
+{
+    public class LoginTests
+    {
+        IWebDriver driver;
+        string user_name;
+        string user_pass;
+
+        [SetUp]
+        public void Setup()
+        {
+            driver = new ChromeDriver();
+            user_name = "admin";
+            user_pass = "123456";
+        }
+
+        [Test]
+        public void Login()
+        {
+            string expected_str = "NAVIGATION";
+
+            driver.Url = "https://localhost/opencart/admin/index.php?route=common/login";
```

Comment:
It could be great to split this variable to two parts - baseUrl = https://localhost/opencart/ and adminUrl = admin/index.php?route=common/login
Using this baseUrl as static property we are able to change all tests and test different environments.